lib/src/app/helpers/createPageLevelParameters.js

```diff
@@ -2,6 +2,12 @@ import defaultSkin from '../skin';
 import defaultsDeep from 'lodash.defaultsdeep';
 import get from 'lodash.get';
 
+/**
+ * Performs complicated merge of default and custom config objects
```

Comment:
Can you expand on "complicated" in the jsdoc?

Reply:
done